api-docs/openapi.json

```diff
@@ -87,9 +87,6 @@
     }
   ],
   "paths": {
-    "/api/0/organizations/{organization_slug}/teams/": {
-      "$ref": "paths/teams/index.json"
-    },
```

Comment:
Path + example JSON can both be deleted as the entire endpoint has been updated

Reply:
this is because we are now generating the docs automatically correct?

Reply:
Yeah it builds the schema automatically so no need to have a JSON file